web/src/beta/features/Editor/index.tsx

```diff
@@ -59,14 +63,37 @@ const Editor: React.FC<Props> = ({ sceneId, projectId, workspaceId, tab }) => {
   const {
     nlsLayers,
     selectedLayer,
+    setSelectedLayerId,
     handleLayerAdd,
     handleLayerDelete,
     handleLayerSelect,
     handleLayerNameUpdate,
+    handleLayerConfigUpdate,
   } = useLayers({
     sceneId,
   });
 
+  const handleLayerSelected = (layerId: string) => {
+    setSelectedLayerStyleId(undefined);
+    handleLayerSelect(layerId);
+  };
```

Comment:
useCallback and maybe move this elsewhere

Reply:
You added the useCallback but I still feel like this and the handleLayerStyleSelected should be in the useLayerStyles and useLayers, respectively